.eslintrc.js

```diff
@@ -1,40 +1,69 @@
+/** @format */
+
 module.exports = {
 	root: true,
-	'extends': [
-		'wpcalypso/react',
-		'plugin:jsx-a11y/recommended',
-		'plugin:jest/recommended',
-	],
+	extends: [ 'wpcalypso/react', 'plugin:jsx-a11y/recommended', 'plugin:jest/recommended' ],
 	parser: 'babel-eslint',
 	env: {
 		browser: true,
 		'jest/globals': true,
+		// mocha is only still on because we have not finished porting all of our tests to jest's syntax
 		mocha: true,
```

Comment:
You can try to turn it off as we most likely migrated all globals to Jest.

Reply:
sure, different PR for that though.